src/main/java/com/example/wbdvsp20bazaarjavaserver/models/User.java

```diff
@@ -0,0 +1,92 @@
+package com.example.wbdvsp20bazaarjavaserver.models;
+
+import javax.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private String username;
+    private String password;
+
+    @Temporal(TemporalType.DATE)
+    private Date dateOfBirth;
+    private String email;
+    private String firstName;
+    private String lastName;
+    private String city;
```

Comment:
super insignificant nit: word choice for city, considering now we are including other regions/continents in the world. it'd be a good idea to decide on a consistent term in our codebase for city/subregion. this doesn't matter too much; the only potential problem i can see is if we look back at this project in the future and get confused for a sec lol.

Reply:
Im just going to keep `city` for now. In the client side, I am just renaming the 'subregion' field to 'city' when it is being processed.